Desktop/data/importers/readstatimporter.cpp

```diff
@@ -113,6 +108,14 @@ ImportDataSet* ReadStatImporter::loadFile(const std::string &locator, boost::fun
 	Log::log() << "Building dictionary" << std::endl;
 	data->buildDictionary(); //Not necessary for opening this file but synching will break otherwise...
 
+
+	Log::log() << "Freeing readstat structs" << std::endl;
+	readstat_parser_free(parser);
+
+#ifdef WIN32
+	io_cleanup();
+#endif
+
```

Comment:
Did you check if this change (freeing stuff after setting the labels) fixes the issue?

Reply:
Of course